src/framework/uicomponents/qml/MuseScore/UiComponents/StyledPopupView.qml

```diff
@@ -123,8 +131,11 @@ PopupView {
                 id: contentBody
                 x: root.margins
                 y: root.margins
-                width: childrenRect.width
-                height: childrenRect.height
+                width: root.contentWidth
```

Comment:
should be enough to set only implicit size
if width or height are not set then implicit size would be used by default

Reply:
Explicit is always better than not explicit :)